protos/local/core_interface.proto

```diff
@@ -102,67 +94,65 @@ message ResolveActivity {
     ActivityResult result = 2;
 }
 
-message CancelTimer {
-    string timer_id = 1;
-}
-
 message UpdateRandomSeed {
     uint64 randomness_seed = 1;
 }
 
 message QueryWorkflow {
-    temporal.api.query.v1.WorkflowQuery query = 1;
+    string query_type = 1;
```

Comment:
```suggestion
    string method = 1;
```

Reply:
I'd like to keep it to type. `method` generally refers to functions with some `self` receiver type -- IE they belong to some class. Different languages might represent different handlers as free-floating functions, whole classes, even maybe a constant (contrived example) so I think type is probably the best thing here.